### DISCUSSION TITLE
beginner help

### DISCUSSION THREAD

**Discussion by ghost** [UPBGE/upbge#1787] [Announcements & others]
Hi

I'm a complete beginner. I need some help to:

- install upbge in Blender 3.4
- i want to make a FPS game and I need example code on how to walk around using the keyboard arrow keys and using the mouse to look around in the world

Thx

**Comment by youle31** [comment]
hi, we try to keep this place for bug reports. I'll convert this to discussion.
Maybe you'd have more luck on our discord (see upbge.org) but I think people generally prefer answering more precise questions (than the second one).

About the first question, there is no installer, you just need to download 1 version of upbge, unzip it and launch blender.exe
